Add READMEs for API content directories (#1895)
github/docs: content/graphql/README.md

# GraphQL

The `/content/graphql` directory is where the GitHub GraphQL API docs live!

> * The `/content/graphql/guides` and `/content/graphql/overview` directories contain regular articles. These are human-editable.

Reviewer: What do you think about a small edit:

```suggestion
* The `/content/graphql/guides` and `/content/graphql/overview` directories contain articles that are human-editable.
```

> * The `/content/graphql/reference` directory contains a skeleton structure for the GraphQL API reference. However, most GraphQL API content inserted using `include`s.

Reviewer: I'm not sure `skeleton structure` is very universal/translatable. What do you think about:

```suggestion
* The `/content/graphql/reference` directory contains an article for each GraphQL data type used in the GitHub API. Most of the content in this directory is rendered using the `include` tag. 
```

>   The content for the `include`s are generated from the API source code internally in GitHub, and is processed using the scripts and static `json` files in the `/lib/graphql` directory.

Reviewer: ```suggestion
  The content rendered by the `include` tag is sourced from the `lib/graphql/static` directory, which is automatically generated and should not be edited by a human. For more information, see the [`lib/graphql/README.md`](lib/graphql/README.md).
```
